ibis/sql/alchemy.py

```diff
@@ -509,6 +509,75 @@ def build_ast(expr, context=None, dialect=None):
     return builder.get_result()
 
 
+class AlchemyDatabaseSchema(Database):
+
+    def __init__(self, name, database):
+        """
+
+        Parameters
+        ----------
+        name : str
+        database : AlchemyDatabase
+        """
+        self.name = name
+        self.database = database
+        self.client = database.client
+
+    def __repr__(self):
+        return "{0}('{1}')".format('Schema', self.name)
```

Comment:
This can be shortened to `'Schema({!r})'.format(self.name)`. We don't need to number fields in Python 2.7 and later.